source/ai/rl/BBF_agent/experience_replay.py

```
@@ -0,0 +1,170 @@
+import torch
+from torch import nn
+import torch.nn.functional as F
+from torch.distributions.categorical import Categorical
```

Ruff (0.12.2)

`torch.distributions.categorical.Categorical` imported but unused

Remove unused import: `torch.distributions.categorical.Categorical`

(F401)